benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataFormat.java

```diff
@@ -16,10 +16,31 @@
 package com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.mysql;
 
 import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.DataFormat;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.Map;
 
 /**
  * Data format for MySQL database.
  */
 public class MysqlDataFormat extends DataFormat {
 
+  private Map<String, LinkedList<String>> tableDataFilesList = new LinkedHashMap<>();
+
+  /**
+   * Add a file to tableDataFilesList.
+   *
+   * @param tablename Table name
+   * @param filename  File name
+   */
+  public void addFile(String tablename, String filename) {
+    if (!tableDataFilesList.containsKey(tablename)) {
+      tableDataFilesList.put(tablename, new LinkedList<>());
+    }
+    tableDataFilesList.get(tablename).add(filename);
+  }
+
+  public Map<String, LinkedList<String>> getTableDataFilesList() {
+    return tableDataFilesList;
```

Comment:
Should this return an Immutable or Unmodifiable version?

Reply:
We want a modified version because values are added to the list at runtime. It depends on the number of rows per table which varies according to the scale factor.

Reply:
It is extremely unintuitive to user a getter to modify the state of the list. You should instead add a method to add tables to the list, rather than modifying the returned list.

Reply:
I misunderstood the question. the list is updated using the addFile method only. will change this to Unmodifiable version.